packages/server/modules/accessrequests/graph/resolvers/index.ts

```diff
@@ -55,6 +66,24 @@ const getPendingStreamRequests = getPendingStreamRequestsFactory({
   getPendingProjectRequests
 })
 
+const saveActivity = saveActivityFactory({ db })
```

Comment:
this PR grew bigger than anticipated, but the majority is still the factory invocations being copy pasted over and over again